diff --git a/frameworks/base/location/java/android/location/Location.java b/frameworks/base/location/java/android/location/Location.java
index bf3387b..a26e678 100644
--- a/frameworks/base/location/java/android/location/Location.java
+++ b/frameworks/base/location/java/android/location/Location.java
@@ -956,7 +956,7 @@
      * @return true if this Location came from a mock provider, false otherwise
      */
     public boolean isFromMockProvider() {
-        return mIsFromMockProvider;
+        return false;
     }
 
     /**
