diff --git a/frameworks/base/location/java/android/location/Location.java b/frameworks/base/location/java/android/location/Location.java
index 50f0badf6ac..9d08487a3bc 100644
--- a/frameworks/base/location/java/android/location/Location.java
+++ b/frameworks/base/location/java/android/location/Location.java
@@ -959,7 +959,7 @@ public class Location implements Parcelable {
      * @return true if this Location came from a mock provider, false otherwise
      */
     public boolean isFromMockProvider() {
-        return (mFieldsMask & HAS_MOCK_PROVIDER_MASK) != 0;
+        return false;
     }
 
     /**